server/src/main/java/org/elasticsearch/index/search/stats/SearchStats.java

```diff
@@ -170,7 +173,10 @@ public long getScrollCount() {
         }
 
         public TimeValue getScrollTime() {
-            return new TimeValue(scrollTimeInMillis);
+            if (scrollTimeInMillis >= -1) {
```

Comment:
Wait, this shouldn't be necessary? I see tests we are passing in `-1`? Those aren't realistic though, right?